index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+
+  <!-- link to fontawesome -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+  <title>Portfolio</title>
+</head>
+<body>
+  <header>
+    <nav class="navbar">
+      <ul>
+        <li class="nav-item logo">Bennyjoez</li>
+        <li class="nav-item logo">
+          <i class="fa-solid fa-bars"></i>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="headline">
+      <span class="greeting">
+        Hey There. <br>
+        I'm Benson
+      </span>
+      <h4 class="title">I'am a Software Developer</h4>
+      <p class="headline-description">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="socials">
+        <div>
+          <img src="./images/github.svg" alt="github">
+        </div> 
+        <div>
+          <img src="./images/Linkedinicon.svg" alt="Linkedin">
+        </div> 
+        <div>
+          <img src="./images/angelist.svg" alt="angelist">
+        </div> 
+        <div>
+          <img src="./images/twitter.svg" alt="twitter">
+        </div> 
+        <div>
+          <img src="./images/medium.svg" alt="medium">
+        </div> 
+      </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! The `list of icons` should have use a `ul` tag to group them, and `li` per icon; each `icon` should have a `link` (`a` tag).  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href 🙏🏼